.circleci/config.yml

```diff
@@ -131,7 +151,7 @@ jobs:
 
   jruby92rails52:
     docker:
-      - image: circleci/jruby:9.2.7.0
```

Comment:
Latest jruby is 9.2.11.0, not sure if CircleCI has yet updated their images though.

Reply:
Good to know! I checked using though rbenv so I missed that.

Looks like CircleCI does have 9.2.11 as I see it here: https://circleci.com/docs/2.0/docker-image-tags.json so good with updating to 9.2.11?

Reply:
Yeah!

Reply:
Ok great! I just pushed an update now. Thanks!